Create adopters list (#97)
open-gitops/project: ADOPTERS.md

# Who uses GitOps? 

> As the GitOps movement grows, we'd like to hear from organizations who are successfully implementing GitOps! Please send a PR with *your* organization name if you are adopting the [GitOps principles](https://opengitops.dev/#principles) in practice. Please include how you are adopting.

Reviewer: I'd change movement to adoption

GitOps isn't a movement in so much that it's a move to a better way to do things.

Reviewer 2: I agree with @chris-short, adoption is a better fit in this case.

Author: Yeah fair point, we can remove that. I do like GitOps as a movement but we don't have to plug it here.

> As the GitOps movement grows, we'd like to hear from organizations who are successfully implementing GitOps! Please send a PR with *your* organization name if you are adopting the [GitOps principles](https://opengitops.dev/#principles) in practice. Please include how you are adopting.

Reviewer 3: I do like the idea that GitOps can be seen as a kind of movement, but I do agree this prob isn't the place to drop that suggestion 🙂

Here's a GitHub code suggestion you can commit directly if you want. Includes what @chris-short said, with small grammar tweaks to accompany that.

```suggestion
As GitOps adoption grows, we'd like to hear from organizations that are successfully implementing GitOps! Please send a PR with *your* organization name if you are adopting the [GitOps principles](https://opengitops.dev/#principles) in practice. Please include how you are adopting.
```

Currently, the following organizations are officially adopting GitOps (in alphabetical order). 

> 1. [Example Organization](https://example.com)

Reviewer 3: What about including a simple example for adopters to describe how they're adopting GitOps?

```suggestion
1. [Example Organization](https://example.com) – `Example` has adopted GitOps to manage our X and Y products and their required infrastructure. We use A and B tools/techniques, and participate in the GitOps WG meetings. See this a (blog post/talk) about the details of our journey and current setup: <https://example.com/blog/how-we-do-gitops>
```
